jenkinsci/tfs-plugin
# Improve references to products such as TFS and Visual Studio Team Services

This pull request essentially removes all references to `VSTS`, including renaming some classes before too many things start depending on sub-optimal names.

Some minor improvements to the documentation were applied at the same time.

Manual testing
--------------
Launched a test Jenkins instance and visited the following areas to confirm the new strings, labels, documentation, etc.:

1. Global configuration:
![image](https://cloud.githubusercontent.com/assets/297515/17113218/aac9a2a0-5277-11e6-884b-96040b61d433.png)
2. Job *Build Triggers*:
![image](https://cloud.githubusercontent.com/assets/297515/17113314/298c7f9a-5278-11e6-84b7-19d978386c6e.png)
3. Job *Build steps*:
![image](https://cloud.githubusercontent.com/assets/297515/17113328/38c62d9e-5278-11e6-90ee-28a8c2dbb512.png)
4. Job *Post-build Actions*:
![image](https://cloud.githubusercontent.com/assets/297515/17113336/45c100be-5278-11e6-8c70-9d7f07235d67.png)
5. Job *SCM* > *Team Foundation Server*:
![image](https://cloud.githubusercontent.com/assets/297515/17113537/35d017ac-5279-11e6-80c5-08dbfc3f8171.png)

I also tried the GIT_CODE_PUSHED and PULL_REQUEST_MERGE_COMMIT_CREATED events at their new endpoint, with success.

Mission accomplished!
